js/js-validated-form.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+   <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+	<title>JS-validated form</title>
+	<script src="validation.js"></script>
+</head>
+<body>
+<h1>JS-validated form</h1>
+
+<form action="./register.php" method="POST" onsubmit="return validate()">
+    <label for="username">Потребителско име</label>
+    <input type="text" id="username">
```

Comment:
You may want to add validations on the html inputs. Giving feedback to the user as early as possible is always better and this will save him time. You could check here for a reference: https://github.com/martinely95/WEB2020-php/blob/master/ex5/index.html